src/native-stack/index.tsx

```diff
@@ -11,6 +11,7 @@ export { default as NativeStackView } from './views/NativeStackView';
 /**
  * Utilities
  */
+export { default as usePreventDismiss } from './utils/usePreventDismiss';
```

Comment:
I'd make this part of the screens package

Reply:
I’m currently using v5. On web, I use stack navigation, and on ios/android I use native-stack.

Should I use this hook only on native, and then use the typical `beforeRemove` event listener in an effect on web?

(Once I upgrade to v6 I assume this won’t matter, but it’s going to take me a little bit.)